x-pack/plugins/apm/readme.md

```diff
@@ -80,19 +80,19 @@ For debugging access Elasticsearch on http://localhost:9220` (elastic/changeme)
 
 ### API integration tests
 
-**Start server**
+**Start server (basic)**
```

Comment:
Does basic refer to these tests being "basic" or that they run using a basic license? I only figured that out after running everything and learning that there are also "trial" tests you can run, so maybe we should point that out.

Reply:
You can probably disregard this since some of it was added in #70050.